test/Android/PrepareEmulator.ps1

```diff
@@ -0,0 +1,86 @@
+[CmdletBinding()]
+Param(
+    [switch]$killRunningEmulator
+)
+
+$xdeProcess = Get-Process | where {$_ -like "*XDE*"}
+if ($xdeProcess.count -gt 0)
+{
+    if ($killRunningEmulator)
+    {
+        <# Kill emulator if already started, otherwise the hanging issue is more likely to repro. #>
+        $xdeProcess | Stop-Process
+    }
+    else 
+    {
+        Write-Output "Emulator already started"
+        exit 0
+    }
+}
+
+$emulatorPath = ${Env:\ProgramFiles(x86)} + '/Microsoft Emulator Manager/1.0'
+
+if (!(Test-Path $emulatorPath))
+{
+    Write-Error "Emulator manager is not installed at $emulatorPath"
+    exit 1
+}
+
+pushd $emulatorPath
+
+$deviceList = ./emulatorcmd list /sku:Android /type:device
+
+<# try to use Lollipop emulator if there is one #>
```

Comment:
I try use the same version of emulator for every run, so we won't randomly use a different version of emulator for each run. And Lollipop arguably is the most used version of Android.